migration_to_v2_0_tests/data/internal_layers_user1.json

```diff
@@ -42,6 +42,28 @@
         "LAYERS": "test_non_existing_internal_layer",
         "FORMAT": "image\/png"
       }
+    },
+    {
+      "className": "OpenLayers.Layer.Vector",
+      "dimensions": {},
+      "legends": [
+        ""
+      ],
+      "maxResolution": null,
+      "metadata": {},
+      "minResolution": 0,
+      "name": "test_migrate_2_user_1:test_vector_layer_sld",
+      "opacity": 1,
+      "path": "group2",
+      "protocol": {
+        "format": "hs.format.WFS",
+        "url": "http://localhost:8000/geoserver/wfs"
```

Comment:
Some test cases to consider:
- WMS, url=`http://localhost:8000/some-client/proxy/geoserver/test_migrate_2_user_1/ows`, layers=`existing_layer_1,nonexistent_layer,existing_layer_2`
- WMS, url=`http://localhost:8000/geoserver/ows`, layers=`existing_ws:existing_layer_1,existing_ws:nonexistent_layer,nonexistent_ws:nonexistent_layer,existing_ws:existing_layer_2`

Reply:
Added as new commit.

Reply:
I am not able to get map thumbnail for the first case, so I removed `nonexistent_layer` from the list.